src/components/copy-button/_copy-button.scss

```diff
@@ -34,7 +34,11 @@
 
     &:before {
       @include layer('overlay');
-      @include typescale('omega');
+      @if not feature-flag-enabled('components-x') {
```

Comment:
Should we split up copy button into the classic/x mixin pattern?

Reply:
I think we should split it up if there are many more of such conditionals - If it's less than a few I think we'd keep a single mix-in.